LicenseOwners/Pages/Index.cshtml

```diff
@@ -7,8 +7,9 @@
 
     <div class="text-center">
         <h1 class="display-4">List of Business Owners</h1>
-        <table border="1">
-            <tr>
+        <table class="table table-striped table-hover" border="10">
+           <thead class="thead-dark">
+            <tr bgcolor="#85929E">
```

Comment:
Good suggestion.  You can improve it by putting this in a linked stylesheet so that it can be centrally managed/maintained, and used across other pages.